### PR TITLE
Upgrade to swiftmailer6

### PR DESCRIPTION
**Pull request by vasike**
#9 

i was able to use this branch in a Symfony 4 which required swiftmailer 6
So maybe a PR for this could help things here to move.

### CODE REVIEW

**Comment by mlelong** [comment]
It would be great to make a new mailjet release, as swiftmailer 6.00 is getting more used, and the current release.

**Comment by Keirua** [comment]
Yup. Any update regarding this PR and the related compatibility issue ?
I'd rather use an official library than writing a custom transport, but seeing the lack of activity in this repository, I'm a bit skeptical it's going to happen...

**Comment by Nightbr** [comment]
ping @maxchampoux, @mailjet/dev-evangelist

**Comment by mskochev** [comment]
Hi guys, thanks a lot for your contributions. We are reviewing all the changes and dependencies at the moment. Hopefully we will be ready to release updates for Swiftmailer, Laravel and Symfony in the next few days.

**Comment by jjsaunier** [comment]
Any news ?

File: composer.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "require": {
         "php": ">=5.4",
```
Comment (gido) [inline composer.json:8]: This can be removed or bumped to `>=7.0.0` ?

**Pull request merged**